### PR TITLE
Evaluation refactor

### PR DESCRIPTION
**Pull request by hannahker**
A lot of commits, but not much major functionality change for review here. This PR accomplishes the following: 

- Updates the three main supported scripts from the workflow (and some notebooks) (```generate_interpolated.py```, ```generate_flood_frac.py```, ```make_plots.py```) to read/write data to the GDrive. 
- Adds QA checks to the above scripts.
- Adds R scripts to summarize results and check the quality of the Gaussian fit.
- Updates documentation as much as possible. 

My main questions for review: 

- Does this structure and general file organization fit with our best practices?
- Is the documentation sufficient to run the main workflow for the pilot evaluation?

### CODE REVIEW

**Review by Tinkaa — COMMENTED** [review]
It looks really great, and would be comfortable merging, only small comments. 
notebooks old data directory sturcture
> Does this structure and general file organization fit with our best practices?

Yes, would say so. Two small things I noticed is that some filenames include capitals, and that some notebooks and R scripts still have paths according to the old data structure. But both are not vital. 
And lets maybe discuss during one of best coding practices how we can best structure config files. 

> Is the documentation sufficient to run the main workflow for the pilot evaluation?

See the few comments, but yes would say so. Let me know if I should try running the workflow with the instructions provided!

**Comment by hannahker** [comment]
Thanks @Tinkaa for the review! Very helpful comments. I'm just going to merge now since I'm having a hard time finding time to address them and they aren't critical at this moment. It would be great to test if you can rerun the whole workflow but that can probably wait for now.

File: analyses/bangladesh/README.md [hunk 1]
```diff
@@ -30,93 +23,50 @@ In addition, the pilot seeks to amplify and coordinate similar anticipatory acti
...
-Set up and activate a Python environment in Anaconda using the ```environment.yml``` file provided: 
-
-```
-conda env create -f environment.yml
-conda activate bang_floods
-```
-
 #### Historical flooding analysis
 
 1. Generate shapefiles that delineate flood extent over time using [this Google Earth Engine Script](https://code.earthengine.google.com/0fe2c1f3b2cf8ef6fe9aa81382b00191). Within the script, the following parameters can be changed: 1) start and end dates of a pre-flood period, 2) start and end dates of a flood period, and 3) shapefile to delineate geographic area of interest.
```
[the remark this answers is not in the document]
Comment (hannahker) [reply analyses/bangladesh/README.md:52]: Yes, good point!

File: analyses/bangladesh/pilot_evaluation/scripts/Generate_flood_frac.py [hunk 2]
```diff
@@ -22,16 +21,17 @@
 # - Column names for shapefiles
 # TODO: Move tests to separate file
 
+DATA_DIR = os.environ['AA_DATA_DIR']
 
-dirs = utils.parse_yaml('pilot_evaluation/config.yml')['DIRS']
-SHP_DIR = dirs['gee_dir']
-OUT_DIR = dirs['data_dir']
+dirs = utils.parse_yaml('analyses/bangladesh/pilot_evaluation/config.yml')['DIRS']
```
[the remark this answers is not in the document]
Comment (hannahker) [reply analyses/bangladesh/pilot_evaluation/scripts/Generate_flood_frac.py:26]: Yeah, that definitely sounds better than the way that it's structured here. I'll raise an issue for that to remind myself to address this in another PR (if you think that it's alright for now).

**Comment by Tinkaa** [comment]
Yes, merging sounds good!

**Pull request merged**